docs/docs/guides/build/io-managers/defining-a-custom-io-manager.md

```diff
@@ -53,7 +53,7 @@ The `upstream_asset` becomes a mapping from partition keys to partition values.
 
 A <PyObject section="partitions" module="dagster" object="PartitionMapping" /> can be provided to <PyObject section="assets" module="dagster" object="AssetIn" /> to configure the mapped upstream partitions.
 
-When writing a custom I/O manager for loading multiple upstream partitions, the mapped keys can be accessed using <PyObject section="io-managers" module="dagster" object="InputContext" method="asset_partition_keys" />, <PyObject section="io-managers" module="dagster" object="InputContext" method="asset_partition_key_range" />, or <PyObject section="io-managers" module="dagster" object="InputContext" method="asset_partitions_time_window" />.
+When writing a custom I/O manager for loading multiple upstream partitions, the mapped keys can be accessed using <PyObject section="io-managers" module="dagster" object="InputContext" />.
```

Comment:
Changing this since it displays kinda odd as the same thing over and over.

<img width="928" alt="Screenshot 2025-06-11 at 7 32 08 AM" src="https://github.com/user-attachments/assets/3b5f7224-c8f6-4473-b4d8-43d999013530" />

We could include the different API docs links in parentheses after the initial `InputContext` but that was looking a little odd as well

Reply:
Oh, that's because there is no `method` property for the `PyObject` component, just `object`. To fix the first one, you would do this:

```
<PyObject section="io-managers" module="dagster" object="asset_partition_keys" displayText="InputContext.asset_partition_keys" />
```

and similar for the others